source/server-discovery-and-monitoring/server-discovery-and-monitoring-logging-and-monitoring.rst

```diff
@@ -111,6 +111,11 @@ Initial Server Description
 
 ``ServerDescription`` objects MUST be initialized with a default description in an “unknown” state, guaranteeing that the previous description in the events and log messages will never be null.
 
+
+Initial Topology Description
+----------------------------
```

Comment:
```suggestion
----------------------------

```

Consistency with other sections, which add a line break. This probably isn't something checked by CI.